code/modules/mob/mob.dm

```diff
@@ -1506,3 +1506,6 @@ mob/proc/assess_threat()
 
 mob/proc/walking()
 	return !(lying || flying)
+
+/mob/proc/dexterity_check()
```

Comment:
I did, could access machines just fine, I guess attack_ghost is exempted of most checks.